### PR TITLE
Selenpy: switch_to_alert

### PR DESCRIPTION
**Pull request by mynguyen2008**

### CODE REVIEW

**Review by trungtranjj — APPROVED** [review]

File: selenpy/support/browser.py [hunk 1]
```diff
@@ -43,3 +47,13 @@ def wait_until(webdriver_condition, timeout=None, polling=None):
         polling = config.poll_during_waits
 
     return wait_for(get_driver(), webdriver_condition, timeout, polling)
+
+def switch_to_alert():
+    try:
+        WebDriverWait(get_driver(),5).until(EC.alert_is_present(),'Timed out waiting for alerts to appear')
+        return get_driver().switch_to.alert
```
Comment (vantruongt2) [inline selenpy/support/browser.py:56]: Why we need to return here?

File: selenpy/support/browser.py [hunk 2]
```diff
@@ -43,3 +47,13 @@ def wait_until(webdriver_condition, timeout=None, polling=None):
         polling = config.poll_during_waits
 
     return wait_for(get_driver(), webdriver_condition, timeout, polling)
+
+def switch_to_alert():
+    try:
+        WebDriverWait(get_driver(),5).until(EC.alert_is_present(),'Timed out waiting for alerts to appear')
```
Comment (vantruongt2) [inline selenpy/support/browser.py]: don't hard timeout here

File: selenpy/support/browser.py (hunk 1 above)
Comment (mynguyen2008) [reply selenpy/support/browser.py:56]: if not return, when using switch_to_alert().text to get text of alert will display error: `AttributeError: 'NoneType' object has no attribute 'text'`

File: selenpy/support/browser.py (hunk 2 above)
Comment (mynguyen2008) [reply selenpy/support/browser.py]: I updated

File: selenpy/support/browser.py [hunk 3]
```diff
@@ -43,3 +47,13 @@ def wait_until(webdriver_condition, timeout=None, polling=None):
         polling = config.poll_during_waits
 
     return wait_for(get_driver(), webdriver_condition, timeout, polling)
+
+def get_alert():
+    try:
+        WebDriverWait(get_driver(),config.timeout).until(EC.alert_is_present())
+        return get_driver().switch_to.alert
+    except UnexpectedAlertPresentException:
+        logging.info("no alert")
```
Comment (vantruongt2) [inline selenpy/support/browser.py:58]: What's happened when the exception is occurred? just log ? is it a returned function?

**Review by vantruongt2 — CHANGES_REQUESTED** [review]

File: selenpy/support/browser.py (hunk 3 above)
Comment (mynguyen2008) [reply selenpy/support/browser.py:58]: I updated

**Pull request merged**